### PR TITLE
Add navbar for vendor profile

### PR DESCRIPTION
**Pull request by neungkl**
Related issue #121 
screen shot 2560-11-21 at 23 31 20

### CODE REVIEW

File: frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<ng-container *appSubNavBar>
```
Comment (futurizing) [inline frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html]: You can combine `ng-container` and `div` tags into a single one:
```html
<div *appSubNavBar class="vendor-bar d-flex">
  ...
</div>
```

File: frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <div class="vendor-item active pt-3 py-2 px-1 mr-4">จัดการตลาด</div>
+    <div class="vendor-item pt-3 py-2 px-1 mr-4">ข้อมูลการเงิน</div>
+    <div class="vendor-item pt-3 py-2 px-1 mr-4">ข้อมูลส่วนตัว</div>
+  </div>
+</ng-container>
+
+<div class="container">
+  <div class="mt-5 mb-3">
+    <h1>โปรไฟล์ผู้เช่า</h1>
+    <div>ข้อมูลของนายอิอิ อุอุ</div>
```
Comment (futurizing) [inline frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html]: Wording like this is discouraged.

File: frontend-webapp/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -28,6 +28,11 @@ const routes: Routes = [
     path: 'search',
     pathMatch: 'full',
     loadChildren: './search/search.module#SearchModule'
+  },
+  {
+    path: 'vendor',
+    pathMatch: 'full',
```
Comment (futurizing) [inline frontend-webapp/src/app/app-routing.module.ts]: As I think there will be a subroute in the future (e.g. `http://localhost:4200/vendor/edit`), consider removing `pathMatch`.

Comment (neungkl) [reply frontend-webapp/src/app/app-routing.module.ts]: Fixed.

File: frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html (hunk 1 above)
Comment (neungkl) [reply frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html]: Fixed.

File: frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html (hunk 2 above)
Comment (neungkl) [reply frontend-webapp/src/app/vendor-dashboard/profile/vendor-profile.component.html]: Fixed.

**Pull request merged**